R/S3_tidy.R

```diff
@@ -81,3 +81,21 @@ tidy.difference_in_means <- tidy_data_frame
 #' @export
 #' @family estimatr tidiers
 tidy.horvitz_thompson <- tidy_data_frame
+
+
+#' @rdname estimatr_tidiers
+#' @templateVar class lh_wrapper
+#'
+#' @export
+#' @family estimatr tidiers
+tidy.lh_robust <- function(x, ...) {
+   do.call(rbind, lapply(x, tidy_data_frame))
```

Comment:
Overly general, will break if the object gets modified and holds a non-tidyable thing. Instead prefer

```
rbind(tidy(x$lm_robust), tidy(x$linear_hypothesis))
```
which is specific and has a clear intent.

Reply:
implemented!